package.json

```diff
@@ -54,12 +54,13 @@
     "prepare": "husky install"
   },
   "devDependencies": {
-    "@types/crypto-js": "^4.2.0",
     "@commitlint/cli": "^19.2.2",
     "@commitlint/config-conventional": "^19.2.2",
     "@types/bun": "^1.1.1",
+    "@types/crypto-js": "^4.2.0",
     "@typescript-eslint/eslint-plugin": "^7.0.1",
     "@typescript-eslint/parser": "^7.0.1",
+    "ai": "^3.1.28",
```

Comment:
Let's remove this.

Reply:
I added that to read the stream in our tests, shall I update the tests to work without ai?

Reply:
Sorry, then lets keep it.